nipype/pipeline/engine/tests/test_engine.py

```diff
@@ -315,7 +315,6 @@ def func2(a):
         name='n2')
     w1 = pe.Workflow(name='test')
     modify = lambda x: x + 1
-    n1.inputs.a = 1
```

Comment:
Not sure if there was a reason for these?

Reply:
nope.